src/models.py

```diff
@@ -226,6 +228,40 @@ def get_task_whitelist(args):
              str(train_task_names), str(eval_clf_names))
     return train_task_names, eval_clf_names
 
+def prepare_elmo_task_specific_skip_embeddings(args, tasks):
```

Comment:
I assume this is mostly repurposed old code?

Reply:
yep!